client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -43,10 +49,10 @@ const PlansColumn = ( { duration, onPlanClick, productType, siteId }: PlanColumn
 					!! product &&
 					duration === product.term &&
 					PRODUCTS_TYPES[ productType ].includes( product.productSlug ) &&
-					// Don't include a generic/option card if the user already owns a subtype
-					! product.subtypes.includes( currentPlan || '' ) &&
 					// Don't include a card of a plan the user already owns
-					product.productSlug !== currentPlan
+					product.productSlug !== currentPlan &&
+					// Don't include a generic/option card if the user already owns a subtype
+					! optionsFromCurrentPlan.includes( product.productSlug )
```

Comment:
`optionsFromCurrentPlan` should be added to the list of deps for `useMemo`.

Reply:
Thanks!